.github/config.yml

```diff
@@ -40,4 +40,13 @@ sentimentBotToxicityThreshold: .9
 # *Required* Comment to reply with
 sentimentBotReplyComment: >
   Please be sure to review the code of conduct and be respectful of other users. // @layer5io/maintainers 
-  
+
+
+
+
+#------------------------------------------------------------------------------------
```

Comment:
Does this work?

This config should be under the #17 stanza, right?

Reply:
You're right @leecalcote , Hi @iambami can you kindly take some time and effect the change and we will be good to go? 😃  The config should l look like this one https://github.com/layer5io/layer5/pull/723/commits/535e1ee3e10d9a6c4bd896f67e703e45ff923749#diff-51ae5ef21b0bd169a283475e122567d3